src/main/flight/gps_rescue.h

```diff
@@ -24,6 +24,7 @@
 typedef struct gpsRescue_s {
     uint16_t angle; //degrees
     uint16_t initialAltitudeM; //meters
+    uint16_t rescueAltitudeBufferM; //meters
```

Comment:
This is in the wrong place, it will have to be appended at the end of the struct: https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Got it. Sorry about that.